extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -65,10 +68,19 @@ class AmpLightbox extends AMP.BaseElement {
       right: 0,
     });
 
+    if (this.isScrollable_) {
+      st.setStyles(this.element, {
+        overflowY: 'auto',
+        overflowX: 'hidden',
```

Comment:
I have a rough example i can include
https://www.evbqa.com/e/test-amp-page-tickets-29711713535/amp - navigate to this page on ios 
STR
1. click read more link under description
2. close lightbox by clicking 'back to listings1
3. click read more link again

The content does not appear at all